docker/update.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+if [ -v PIZZA_UP ]
+then
+	# update repo
+	cd /opt/pizza-sync
+	git pull --rebase
+
+	# rebuild frontend
+	cd frontend
+	yarn && yarn run build:prod
```

Comment:
what's the point of building it once in the dockerfile if you do it again here? I think that you either need to follow my recommendation above (do not try to upgrade the application) or build everything here all the time and not in the dockefile (bad idea imho)